src/libraries/System.IO.Compression.Brotli/src/System.IO.Compression.Brotli.csproj

```diff
@@ -2,6 +2,7 @@
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-unix;$(NetCoreAppCurrent)</TargetFrameworks>
+    <UseIntellisenseDocumentationFile>true</UseIntellisenseDocumentationFile>
```

Comment:
This property's name doesn't really help you understand what it does. I think something like `UseIntellisensePackageDocumentationFile` would make more sense.

Reply:
Should we use the reserved one instead? https://learn.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#generatedocumentationfile

Reply:
Probably not as we always want to generate documentation files regardless of whether they are used or not.

Reply:
It's also not clear to me what you actually want he behavior to be here.  I see these projects set this to `true` and setting to `true` causes the private.intellisense package to be used -- which I thought was the default?

Reply:
Agreed. Let's default `UseIntellisensePackageDocumentationFile` to true in the intellisense.targets file (if not previously set) and make these project files set it to false instead.

Reply:
Ok sure.

Reply:
> Let's default `UseIntellisensePackageDocumentationFile` to true in the intellisense.targets file (if not previously set)

Ah, I missed the "if not previously set" part.